CODE/bc_p.f90

```diff
@@ -545,21 +553,21 @@ SUBROUTINE APPLY_BOUNDARY(N,XPER,YPER,ZPER,IPERIODICITY,XMPIELRANK)
 end do
 jj1=0
 !$OMP DO SCHEDULE (STATIC)
-DO I=1,KMAXE			!ALL ELEMENTS
-    if (ielem(n,i)%interior.eq.1)then		!THAT HAVE AT LEAST ONE UNKNWON NEIGHBOUR
-	    IF (ielem(n,i)%nofbc.GT.0)THEN		!THAT HAVE AT LEAST ESTABLISHED A BOUNDARY CONDITION CODE
-		  DO J=1,ielem(n,i)%IFCA			!LOOP ALL THEIR BOUNDARY FACES
+DO I=1,KMAXE			!> ALL ELEMENTS
```

Comment:
is doxygen convention to have inline comments rather than top of the function/subroutine ?